topics/introduction/slides/introduction.html

```diff
@@ -238,32 +312,47 @@
 - Create **sub**-workflows: a workflow inside another workflow
 - **Share** workflows for publication and with the community
 
+???
+
+Potential information overload for newbies
+
 ---
 
-### Data sharing
+### Visualizations
+
+.image-75[![](../../dev/images/charts_examples2.png)]
 
-- You can share your Galaxy items - histories, workflows, visualizations, and pages - with other people in three different ways:
+- Datatypes know what tools can be used to visualize them:
+  - Sequencing data has a button for visualizing in IGV
+  - Tabular data will prompt you to build charts
+  - Protein data can be seen in a 3D viewer
+- Interactive environments: Jupyter, RStudio, etc
+
+---
+
+### Shared data
+
+- Share everything you do in Galaxy - histories, workflows, and visualizations
   - Directly using a Galaxy account's email addresses on the same instance
   - Using a web link, with anyone who knows the link
   - Using a web link and publishing it to make it accessible to everyone from the *Shared Data* menu
 
+---
 
-- Tools are shared using the free tool store: [Galaxy Tool Shed](https://toolshed.g2.bx.psu.edu/)
+### Community
 
----
+Be part of an active and friendly community
 
-### Data visualization
+- Support and Questions: [Galaxy Help](https://help.galaxyproject.org/)
 
-- Charts
-- Each datatype can have some visualizations associated
-- Track browser called Trackster
-  - To visualize genomic data in a tightly integrated way
+  ![Galaxy Help](../images/galaxy_help.png)
 
----
+- Community curated documentation: [Galaxy Community Hub](https://www.galaxyproject.org/)
+- Training around Galaxy for scientists, developers, admins, instructors: [Galaxy Training Community]({{site.url}}/{{site.baseurl}}). Chat with training community on [Gitter](https://gitter.im/Galaxy-Training-Network/Lobby)
```

Comment:
I'm not sure about the chat link here. Do we want to be the first point of contact??

Maybe we need a chat directory to point people to the right channel.

Reply:
How else should people who wish to contact people associated with training find us? It is not meant to push GTN as entry point, the focus is on Galaxy Help which is above + has a giant picture. I can maybe find a way to clarify in not too many words, "if you want to discuss training materials, here is our chat"